linbox/blackbox/submatrix.h

```diff
@@ -377,17 +377,17 @@ namespace LinBox
 
 		size_t rowfirst() const
 		{
-			return this->_r0;
+			return 0;
```

Comment:
because BlasSubmatrix does not hold anymore a hand on the original matrix, only the data through the pointer is kept. 
By the way I do not see the usefulness of rowfirst and colfirst in the API, it is never used in the lib., should we remove it ?

Reply:
OK, seems better to remove it then.

Reply:
fine with me

Reply:
Hum it is actualy used in SubmatrixOwner, see next comment ?

Reply:
Ok we decided to keep it. In particular, it helps to construct another similar submatrix, for instance in rebind (take the original blackbox, rebind and take a submatrix of it).

For BlasSubmatrix specialisation we kept firstrow() and firstcol() to be 0. The reason is we do not have a pointer to the original matrix. Therefore the getPtr() method only returns a pointer to the current BlasSubmatrix and the value to construct a similar submatrix from itself is to use the principal submatrix.